spk/flexget/src/requirements.txt

```diff
@@ -1,51 +1,55 @@
 # Produced by pip freeze
 # Commented if provided by python package at /usr/local/python/share/wheelhouse/
-aniso8601==1.2.0
-APScheduler==3.3.1
+aniso8601==1.2.1
+apscheduler==3.5.0
```

Comment:
To ease future update maintenance, we recommend to document requirement.txt file generation in Makefile.
As an example: https://github.com/SynoCommunity/spksrc/blob/master/spk/borgbackup/Makefile#L11-L16

Reply:
Don't see reason to do that. The requirements.txt for 99% is a copy of https://github.com/Flexget/Flexget/blob/develop/requirements.txt.
Just added flexget and transmissionrpc (I found that it must be installed if user want to enable transmission plugin in the flexget) packages.

Reply:
For maintenance I recommend to document inside requirements.txt (and not in the Makefile) where the content is taken from.
@manowark I propose to remove the comment `# Produced by pip freeze` and to add what you commented here. This should include the date (or commit hash) of the original file in the flexget repo.

Reply:
@hgy59 it's a good idea. I added link to tag.